automatic/zotero/update.ps1

```diff
@@ -1,42 +1,36 @@
 ﻿Import-Module Chocolatey-AU
 Import-Module "$PSScriptRoot\..\..\scripts\au_extensions.psm1"
 
-$releases = 'https://www.zotero.org/download/client/dl?channel=release&platform=win32'
-$softwareName = 'Zotero'
+$releases = 'https://www.zotero.org/download/client/dl?channel=release&platform=win-x64'
 
 function global:au_BeforeUpdate { Get-RemoteFiles -Purge -NoSuffix }
 
 function global:au_SearchReplace {
-  $version = $Latest.Version.ToString()
-
   @{
     ".\legal\VERIFICATION.txt" = @{
-      "(?i)(\s*1\..+)\<.*\>" = "`${1}<$($Latest.URL32)>"
-      "(?i)(^\s*checksum\s*type\:).*" = "`${1} $($Latest.ChecksumType32)"
-      "(?i)(^\s*checksum(32)?\:).*" = "`${1} $($Latest.Checksum32)"
+      "(?i)(^\s*x32\:).*" = "`${1} $($Latest.URL32)"
+      "(?i)(^\s*x64\:).*" = "`${1} $($Latest.URL64)"
+      "(?i)(^\s*checksum32\:).*" = "`${1} $($Latest.Checksum32)"
+      "(?i)(^\s*checksum64\:).*" = "`${1} $($Latest.Checksum64)"
     }
     ".\tools\chocolateyInstall.ps1" = @{
-      "(?i)^(\s*softwareName\s*=\s*)'.*'" = "`${1}'$softwareName'"
-      "(?i)(^\s*file\s*=\s*`"[$]toolsPath\\).*" = "`${1}$($Latest.FileName32)`""
-    }
-    ".\tools\chocolateyUninstall.ps1" = @{
-      "(?i)^(\s*softwareName\s*=\s*)'.*'" = "`${1}'$softwareName'"
-    }
-    ".\$($Latest.PackageName).nuspec" = @{
-      "(\<releaseNotes\>).*(\<\/releaseNotes\>)" = "`${1}https://www.zotero.org/support/$($version.Major).$($version.Minor)_changelog`${2}"
+      "(?i)(^[$]File32Name =).*" = "`${1} '$($Latest.URL32.split('/')[-1])'"
+      "(?i)(^[$]File64Name =).*" = "`${1} '$($Latest.URL64.split('/')[-1])'"
     }
   }
 }
 
 function global:au_GetLatest {
-    $url = Get-RedirectedUrl -url $releases
+   $url64 = Get-RedirectedUrl -url $releases
+   $url32 = $url64 -replace 'x64','win32'
 
-    $version  = $url -split '/' | Select-Object -Last 1 -Skip 1
+   $version  = $url64 -split '/' | Where-Object {$_ -match '^\d+\.\d[0-9.]*$'} | Select-Object -Last 1
 
-    @{
-        Version      = $version
-        URL32        = $url
-    }
+   @{
+      Version      = $version
+      URL32        = $url32
+      URL64        = $url64
+   }
 }
 
-update -ChecksumFor none
+update -ChecksumFor none -force
```

Comment:
Force should not be hardcoded into the update script.

Reply:
Oops!  Fixed now.